### ISSUE TITLE
Host Packs / Partner Packs:  Get Localized versions LIVE on the MP site

### ISSUE THREAD

**Issue by ScottDowne** [MozillaFoundation/Advocacy#499]
From: https://github.com/MozillaFoundation/Advocacy/issues/452

We have a few links that to me need to be updated with localized content, if possible.

Examples: the partner pack and host pack links.

cc @damiankay @ldecoursy @HPaulJohnson

**Comment by valianttry** [comment]
@ScottDowne I have host pack localized, but I don't know 1) correct format (PDF or HTML) and 2) where will they be hosted / what will the link to each of them be?

Similar to:
#498

**Comment by valianttry** [comment]
@ScottDowne I have the host pack localized. Localizations are here:

[Maker Party Host Pack_Translations.zip](https://github.com/MozillaFoundation/Advocacy/files/499742/Maker.Party.Host.Pack_Translations.zip)

**Comment by ScottDowne** [comment]
@valianttry how hard would it be to move these into google docs? So a google doc for each locale?

Updating the links in learning is pretty easy, I just need to figure out the best way to host them, and I'm thinking google doc has better support and is easier for us to maintain and update. (the current english one is on google docs too)

**Comment by ScottDowne** [comment]
@valianttry and DE and FR look to have strike out lines in them. Example:

![strike](https://cloud.githubusercontent.com/assets/197334/18957109/d29909b4-862b-11e6-867a-95f2944a5535.png)

It looks like people are still working on them? Or was it just the strikeouts didn't get removed yet? I'm not super familiar with the process of localizing a doc, though.

**Comment by valianttry** [comment]
@ScottDowne turn off track changes. I don't see any of those because I have track changes turned off. So, in Word, uncheck all these boxes:

![highlight_changes_and_emails_batch_2_de_docx__compatibility_mode_](https://cloud.githubusercontent.com/assets/9830371/18960358/4e07645a-861e-11e6-9b0e-6cdc5f68e21f.png)

**Comment by ScottDowne** [comment]
Ah, makes sense. (I cannot use word in Linux, but I found the equivalent.)

Still hoping we can get these into google docs to simplify the hosting and maintenance. (So I don't need to be pinged to make updates)

**Comment by ScottDowne** [comment]
@HPaulJohnson @ldecoursy @valianttry Thoughts on putting these into google docs? (more info in the comments above.)

**Comment by ldecoursy** [comment]
Works for me.  I know @lovegushwa also did a version of the English host pack in Google slides. @shaghdoosti do you have a preference?

**Comment by ScottDowne** [comment]
Just updating this, based on conversations from #352, to make sure we're all on the same page.

@ldecoursy mentioned that the host pack localizations from the zip files, are being moved into  google slides:
- ES https://docs.google.com/a/mozillafoundation.org/presentation/d/12hDy1vnd73GBRXUxiFIN_sr7lJ8e_5k3vtQfYW5SfdI/edit?usp=xpa_n&ts=57ee91f2
- DE https://docs.google.com/a/mozillafoundation.org/presentation/d/1Or0fja5tUXGONvVEI9IiH772t2Rss1EoRDeCX32syzI/edit?usp=xpa_n&ts=57ee85a0

@ldecoursy Is the data being retrieved from the zip file @valianttry linked to in an above comment? This was news to me, so I just wanted to make sure work wasn't being duplicated.

**Comment by ScottDowne** [comment]
@ldecoursy and what about partner packs?

**Comment by ldecoursy** [comment]
@ScottDowne that's correct. Is this the best place to link to the other localized host packs (in Google Slides format) when they're completed?

**Comment by ldecoursy** [comment]
@ScottDowne I haven't seen the localized versions of that. The ENG is here https://docs.google.com/presentation/d/1Ibb0WgTwdXBON1UxYjQMu3DftK2PcwNiiCCKgkuibZs/edit#slide=id.g165eabf55d_2_173

**Comment by ScottDowne** [comment]
Yeah, any content being finished around host/partner packs and localization should be in here.

**Comment by ldecoursy** [comment]
All localized host packs in the Google Slides template are here: https://drive.google.com/drive/folders/0B9eLa2Nihrhgcno3bm14NGpoX0E

Thanks @KGorr!

cc @ScottDowne

**Comment by ScottDowne** [comment]
@ldecoursy I don't have access to view those, I requested it though.

**Comment by ScottDowne** [comment]
I need individual links, and they need to be viewable by the public.

**Comment by valianttry** [comment]
See #471

**Comment by ldecoursy** [comment]
- Maker Party Host Pack - PL https://docs.google.com/presentation/d/1bUkd1zWFJdzitS2i7ykWsF5xteFcrXAy6W2IhOJlXH0/edit?usp=sharing
- Maker Party Host Pack - DE https://docs.google.com/presentation/d/1Or0fja5tUXGONvVEI9IiH772t2Rss1EoRDeCX32syzI/edit?usp=sharing
- Maker Party Host Pack - EN https://docs.google.com/presentation/d/1OpV62j95tN-CuRy2RGWuKXw1P1OS9Vs-94DzkjXS8Po/edit?usp=sharing
- Maker Party Host Pack - FR https://docs.google.com/presentation/d/1BG7qk2HvSgIax2oaFKXThT_jEjdh8YfeHeTeVZjxDdk/edit?usp=sharing
- Maker Party Host Pack - IT https://docs.google.com/presentation/d/15Rkio1xrM4I3H20OhQZUgArEtxWJIJWAFScvT0H9Dv0/edit?usp=sharing
- Maker Party Host Pack - ES https://docs.google.com/presentation/d/12hDy1vnd73GBRXUxiFIN_sr7lJ8e_5k3vtQfYW5SfdI/edit?usp=sharing

**Comment by ScottDowne** [comment]
@ldecoursy so the host packs are about to land, I'm still not sure what's happening with the partner packs. Although now that I look at the partner pack, I think there was a mixup.

Thoughts? This is the partner pack: https://drive.google.com/file/d/0B3P4kAmkrxtCXzVMZG5qUDVhX3c/view

But that looks like the new host packs.

**Comment by ScottDowne** [comment]
@ldecoursy If this isn't as you expected, I think we should sort this out on vidyo. As I think you're actually giving me partner packs, and Andrea was giving me host packs in a comment way at the start.

**Comment by ScottDowne** [comment]
OK, I confirmed, these are in fact host packs, they just share the partner pack designs.

Still waiting on localized partner packs.

**Comment by ScottDowne** [comment]
The host pack localized links are live on learning staging.

I was going to do both host packs and partner packs in this ticket, but I suspect a separate ticket for each would have probably been easier?

Either way, keeping this open until we have partner packs.

**Comment by ldecoursy** [comment]
@ScottDowne Does that mean the current link to the Host Pack at https://learning.mozilla.org/en-US/events/resources will be updated (currently it's linking to a google doc with some comments, not the google slide version). Or should I file a separate ticket for that?

**Comment by ScottDowne** [comment]
Yes, it'll work once it's live. Right now it's only on staging: https://learning.mofostaging.net/en-US/events/resources

**Comment by valianttry** [comment]
Confirmed that Partner Packs do not need to be localized. Please go ahead and post the EN for Partner Pack. 

For future yes please separate these kinds of tasks into separate tickets; 🚀 💯

**Comment by ScottDowne** [comment]
Cool, so I think then if en-US partner pack is sufficient, we should close this, as the rest is on staging and that should be enough to resolve this.

**Comment by valianttry** [comment]
@ScottDowne when will it be pushed to prod?

**Comment by ScottDowne** [comment]
@valianttry When we're ready to ship all of maker party is probably best. Today, for sure.

**Comment by ScottDowne** [comment]
OK, this is done and live. :rocket: :roller_coaster: :robot:
